docs/install.rst

```diff
@@ -21,7 +21,13 @@ Linux + Docker
 OS X + Docker
 -------------
 
-1. Install Docker_ (via Boot2Docker_).
+1. Install Docker_ (via the Toolbox Instructions_).
+2. Run the Docker Machine & load its environment
+
+       $ docker-machine start default
```

Comment:
There are no machines available by default; AFAIK there is nothing special about `default` name